app/assets/stylesheets/_settings.scss

```diff
@@ -48,22 +48,22 @@
 
 $global-font-size: 100%;
 $global-width: rem-calc(1200);
-$global-lineheight: 1.5;
+$global-lineheight: 1.4;
 $foundation-palette: (
   primary: #2296A0,
   secondary: #FBC352,
   success: #3adb76,
   warning: #ffae00,
   alert: #cc4b37,
 );
-$light-gray: #efefef;
-$medium-gray: #cacaca;
-$dark-gray: #8a8a8a;
+$light-gray: #F9FAFA;
+$medium-gray: #838D91;
+$dark-gray: #3E4E54;
```

Comment:
Maybe we could do something like this to abstract colours out https://github.com/unepwcmc/grasp/blob/master/app/assets/stylesheets/variables/colors.scss